lighthouse-core/audits/timing-budget.js

```diff
@@ -89,23 +89,47 @@ class TimingBudget extends Audit {
    * @return {Array<BudgetItem>}
    */
   static tableItems(budget, summary) {
+    /** @type {Array<BudgetItem>} */
+    const items = [];
     if (!budget.timings) {
-      return [];
+      return items;
     }
-    return budget.timings.map((timingBudget) => {
+
+    for (const timingBudget of budget.timings) {
       const metricName = timingBudget.metric;
       const label = this.getRowLabel(metricName);
       const measurement = this.getMeasurement(metricName, summary);
       const overBudget = measurement && (measurement > timingBudget.budget)
         ? (measurement - timingBudget.budget) : undefined;
-      return {
-        metric: metricName,
-        label,
-        measurement,
-        overBudget,
-      };
-    }).sort((a, b) => {
-      return (b.overBudget || 0) - (a.overBudget || 0);
+
+      if (metricName === 'cumulative-layout-shift') {
```

Comment:
It's unfortunate how complex this has made this fn + the sort at the end.

wb ~L167, where `tableItems` is called, we filter the results and replace the CLS measurement value instead ?